presentation/src/main/java/com/knocklock/presentation/lockscreen/StartApplicationService.kt

```diff
@@ -0,0 +1,92 @@
+package com.knocklock.presentation.lockscreen
+
+import android.app.Notification
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.app.PendingIntent
+import android.app.PendingIntent.FLAG_IMMUTABLE
+import android.app.Service
+import android.content.Context
+import android.content.Intent
+import android.content.IntentFilter
+import android.os.Build
+import android.os.IBinder
+import androidx.core.app.NotificationCompat
+
+/**
+ * @Created by 김현국 2022/12/13
+ * @Time 4:09 PM
+ */
+class StartApplicationService : Service() {
+
+    var receiver: StartApplicationReceiver? = null
+    private val ANDROID_CHANNEL_ID = "KnockLockScreenNotification"
+    private val ANDROID_CHANNEL_NAME = "KnockLockScreen"
+    lateinit var notificationManager: NotificationManager
```

Comment:
```suggestion
    private lateinit var notificationManager: NotificationManager
```

여기도 private로 해줘도 될 것 같아요!

Reply:
@jaeryo2357 [bfd8ac0](https://github.com/Knock-Lock/Knock-Lock/pull/61/commits/bfd8ac0f90ffec3f45de34babb17511035706702) 반영했습니당